crates/anvil/src/eth/fees.rs

```diff
@@ -204,11 +206,18 @@ pub struct FeeHistoryService {
 
 impl FeeHistoryService {
     pub fn new(
+        spec_id: SpecId,
```

Comment:
should we store the blobparams here instead?

Reply:
Fixed in https://github.com/foundry-rs/foundry/pull/10565/commits/d4a76cbceb2edbaa236e86d19be8e2e90c422d46